Cargo.toml

```diff
@@ -38,6 +40,7 @@ lazy_static = "1.0.0"
 rustwide = "0.7.1"
 mime_guess = "2"
 dotenv = "0.15"
+git2 = { version = "0.13.6", default-features = false }
```

Comment:
We already have git2 through `crates-index-diff`, why add it again here?

Reply:
We use it directly in `load_config`.

Reply:
You can `use crates_index_diff::git2`, they re-export it. That prevents the versions getting out of sync and accidentally bundling two.

Reply:
Huh, what's with all these re-exports, that's not something I even thought to look for, I only noticed `iron::url` did a re-export since it was already used that way in some locations.

The versions can't get out of sync since cargo will refuse to include two semver compatible versions in the build, but I'll change to use it.

Reply:
> You can use crates_index_diff::git2, they re-export it. That prevents the versions getting out of sync and accidentally bundling two.

I don't think this is a good approach, if docs.rs depends on git2 it should do so explicitly.